### ISSUE TITLE
Kubuntu 15.04 + keepass2-plugin-application-indicator dependency problem

### ISSUE THREAD

**Issue by tadamic** [dlech/Keebuntu#25]
Hi, 

I'm using new Kubuntu, packaged keepass2 and trying to install like this:

~~~
sudo apt-get install keepass2-plugin-application-indicator
...
The following packages have unmet dependencies:
 keepass2-plugin-application-indicator : Depends: keepass2-plugin-libimagemagick (= 0.2.7+0.4.8) but it is not going to be installed
~~~

Then this:

~~~
sudo apt-get install keepass2-plugin-libimagemagick
The following packages have unmet dependencies:
 keepass2-plugin-libimagemagick : Depends: libmagickwand5 but it is not installable
~~~

```libmagickwand5``` package is no longer available in 15.04 as can be seen here: https://launchpad.net/ubuntu/vivid/amd64/libmagickwand5/8:6.7.7.10+dfsg-4ubuntu1 (status: Deleted)

Could you update dependencies?

Thnx

**Comment by dlech** [comment]
I already fixed this in #23. Make sure you are using the beta ppa.

**Comment by tadamic** [comment]
Yes, the one from beta works flawlesly, thnx and sorry for the duplicate issue.
